src/main/kotlin/no/nav/syfo/dialogmote/database/MoteQuery.kt

```diff
@@ -104,9 +104,10 @@ fun Connection.createNewDialogmoteWithReferences(
         commit = false,
         moteId = moteId,
         personIdentNumber = newDialogmote.arbeidstaker.personIdent,
+        fritekstInnkalling = newDialogmote.arbeidstaker.fritekstInnkalling
     )
     this.createMotedeltakerArbeidsgiver(
-        commit = true,
+        commit = false,
```

Comment:
@janzh Så vidt jeg kan skjønne bør parameterveriden på dette og de andre kallene nedenfor være false, fordi man skal gjøre commit for alt under ett dersom funksjonen som helhet kalles med commit=true. Eller er det noe jeg har misforstått her?